dart/test/hub_test.dart

```diff
@@ -170,7 +170,7 @@ void main() {
         bindToScope: true,
       );
 
-      hub.configureScope((Scope scope) {
+      await hub.configureScope((Scope scope) {
```

Comment:
I'd add a new test that `hub.configureScope` also takes a non-async method.

Reply:
Not sure I understand. The one here is a non-async method as param.

Reply:
Sorry, I meant, we should add a test that tests an async callback for `configureScope`.
E.g.

```
      await hub.configureScope((Scope scope) async {
        // do something with async work because the method is `async`
      });
```

Reply:
It could be .e.g. `await Future.delayed(Duration(seconds: 1));` but with just a few millis.